<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGroup</class>
 <widget class="QWidget" name="addGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>180</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 5</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>36</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New group:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>296</y>
     <width>153</width>
     <height>86</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 7</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>66</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 3</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>66</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 1</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>180</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 6</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 4</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_groupNum">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>36</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>***-***</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>66</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 2</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>426</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>296</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 8</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_16">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>296</y>
     <width>153</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Student 9</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLineEdit" name="lineEdit_17">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_18">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>surname</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
